Update documentation for v2.2 (#162)
ServiceNow/azimuth: docs/docs/user-guide/exploration-space/confusion-matrix.md

# Confusion Matrix

The confusion matrix displays the **model confusion between each pair of intents**. The confusion is
defined as the number of utterances with a given label that are predicted as another label.

The toggle "Normalize" in the top right corner allows alternating between normalized and raw values.

> When normalized, the number of utterances are divided according to the total number of utterances

Reviewer: "is divided by"?
```suggestion
When normalized, the number of utterances is divided by to the total number of utterances
```
